### PR TITLE
Feat/add react native expo client

### PR DESCRIPTION
**Pull request by thorsten-stripe**

### CODE REVIEW

File: custom-payment-flow/client/react-native-expo/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  try {
+    const response = await fetch(`${API_URL}/config`);
+
+    const {publishableKey} = await response.json();
+
+    return publishableKey;
+  } catch (e) {
+    console.log(e);
+    console.warn('Unable to fetch publishable key. Is your server running?');
+    Alert.alert(
```
Comment (cjavilla-stripe) [inline custom-payment-flow/client/react-native-expo/helpers.ts:14]: Nice!

**Review by cjavilla-stripe — COMMENTED** [review]
Super cool. Thanks for adding! Is it expected that I see this after authorizing the test payment on for the 3155 SCA card? Guessing it's related to redirecting back into the app or something.

image

**Comment by cjavilla-stripe** [comment]
LGTM

**Comment by thorsten-stripe** [comment]
> Super cool. Thanks for adding! Is it expected that I see this after authorizing the test payment on for the 3155 SCA card? Guessing it's related to redirecting back into the app or something.

Fixed via https://github.com/stripe-samples/accept-a-payment/pull/123/commits/150b7e7dceccd9d7c151b2ac89a9c7e257f09c7c

**Pull request merged**